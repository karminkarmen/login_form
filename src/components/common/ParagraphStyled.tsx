import styled from 'styled-components';

const ParagraphStyled = styled.p`
    margin: 0 0 ${props => props.theme.margins.normal} 0;
    max-width: ${props => props.theme.width.innerElementWidth};
    color: ${props => props.theme.colors.main};
    font-size: ${props => props.theme.fontSizes.normal};
    font-family: ${props => props.theme.fontFamilies.lato};
`;

export default ParagraphStyled;