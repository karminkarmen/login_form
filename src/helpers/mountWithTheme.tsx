import React from 'react';
import { mount } from 'enzyme';
import { ThemeProvider } from 'styled-components';
import theme from '../configs/theme';

const mountWithTheme = (child: React.ReactElement) => {
    return mount(child, {
        wrappingComponent: ({ children }) => <ThemeProvider theme={theme}>{children}</ThemeProvider>,
    });
}

export default mountWithTheme;