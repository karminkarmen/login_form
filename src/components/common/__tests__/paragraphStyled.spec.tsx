import React from 'react';
import mountWithTheme from '../../../helpers/mountWithTheme';
import ParagraphStyled from '../ParagraphStyled';

describe('<ParagraphStyled />', () => {

    it('matches snapshot', () => {
        const wrapper = mountWithTheme(<ParagraphStyled />);
        expect(wrapper.debug()).toMatchSnapshot();
    });

});
