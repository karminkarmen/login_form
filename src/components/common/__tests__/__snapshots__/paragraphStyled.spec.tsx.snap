// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ParagraphStyled /> matches snapshot 1`] = `"<p class=\\"sc-AxjAm jpfrvO\\"></p>"`;
